workshop/content/exercises/pipeline-resources.adoc

```diff
@@ -1,15 +1,15 @@
-Before we can trigger the `pipeline` just created, `pipelineresources` must be
-defined as inputs and outputs for the build `task` of the `pipeline`.
+Before we can trigger the pipeline just created, pipelineresources must be
+defined as inputs and outputs for the `build` task of the pipeline.
 
-The build `task` of your `pipeline` takes a git repository as an input and then
-produces an image that is pushed to an image registry. `PipelineResources` are how
+The `build` task of your pipeline takes a git repository as an input and then
+produces an image that is pushed to an image registry. Pipelineresources are how
```

Comment:
This one is a bit fuzzy. IMO, and feel free to disagree, but if you're referring to:

- The Kubernetes resource type itself, such as in a code snippet, use the actual kind name PipelineResource (which I believe is the way it's capitalized, but I might be wrong).

- The concept itself, which I'd argue is the usage here, should use the more user-friendly words for it. It's debatable if it's a formal noun and thus "Pipeline Resources" or should simply be "Pipeline resources".

Reply:
Went with separating the words here (i.e. pipeline resources) except for a few references that are focused on the actual crd